home02/templates/hello.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h1>Привет, {{ name }}</h1>
+<form action="{{ url_for('del_cookie') }}"
+      method="post">
```

Comment:
Если сделать как я сказал выше, то атрибут action будет не нужен.
```suggestion
<form method="post">
```